v2g_liberty.py

```diff
@@ -124,29 +124,31 @@ def restart_charger(self, *args, **kwargs):
         self.set_charger_action("restart")
         self.notify_user("Restart of charger initiated by user. Please check charger.")
 
-    # TODO: combine with same function in other modules??
-    def notify_user(self, message: str, critical=False):
+    # ToDo: Make generic function in utils? See get_fm_data.py for equivalent.
+    def notify_user(self, in_message: str, critical=False, in_title=""):
```

Comment:
`message` is a clearer variable name then `in_message`.

Reply:
`title` is a clearer variable name then `in_title`. It would be better typed as `title: Optional[str] = None`.

Reply:
`from typing import Optional`

Reply:
Good suggestion, will do.